modules/mod_admin/templates/_admin_view_types.tpl

```diff
@@ -0,0 +1,7 @@
+{% block content_types %}
+    {% for content_type, url in id|content_type_urls %}
```

Comment:
Ok, separated the page dispatch rule. I still prefer to call it ‘page’ over ‘rendered page’ as the term ‘rendered’ doesn’t mean so much to end users.